spec/javascripts/components/story/story_link_spec.js

```diff
@@ -0,0 +1,72 @@
+import jasmineEnzyme from 'jasmine-enzyme';
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import StoryLink from 'components/stories/StoryLink';
+import Icon from 'components/shared/Icon';
+
+import Story from 'models/story.js';
+
+const story = {
+  title: 'Story 2',
+  created_at: sinon.stub(),
+  get: sinon.stub(),
+  humanAttributeName: sinon.stub().returns('Description'),
+  escape: sinon.stub().returns('description'),
+  hasNotes: sinon.stub().returns(false),
+  views: [{highlight: sinon.stub()}]
+};
+
+story.get.withArgs('id').returns('2');
+story.get.withArgs('requested_by_name').returns('Test');
+story.get.withArgs('state').returns('unscheduled');
+story.get.withArgs('story_type').returns('feature');
+
+describe('<StoryLink />', function() {
+
+  beforeEach(function() {
+    jasmineEnzyme();
+    I18n = {t: sinon.stub()};
+    window.md = {makeHtml: sinon.stub()};
+    document.getElementById = sinon.stub().returns({scrollIntoView: sinon.stub()})
+  });
+
+  afterEach(function() {
+    sinon.restore(I18n);
```

Comment:
This won't work, you're overwriting those objects yourself, you didn't do something like: `sinon.stub(I18n, 't')`.

You can do like this (for both I18n, md and document.getElementById):

```js
sinon.stub(I18n, 't');

// ...
I18n.t.restore();
```